test/unit/lib/ActivityStream.test.js

```diff
@@ -255,7 +255,7 @@ describe("ActivityStream", () => {
     it("should be false with expected geo and unexpected locale", () => {
       sandbox.stub(global.Services.prefs, "prefHasUserValue").returns(true);
       sandbox.stub(global.Services.prefs, "getStringPref").returns("US");
-      sandbox.stub(global.Services.locale, "getAppLocaleAsLangTag").returns("no-LOCALE");
+      sandbox.stub(global.Services.locale, "appLocaleAsLangTag").get(() => "no-LOCALE");
```

Comment:
slightly different than the backport, since these are getters now the stubs needed updating in order for our tests to pass